docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +83,155 @@ metadata:
+* `hap.clusterRegionRule` - if evaluated to true the `_<HYPERSCALER_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as cluster region based search.
+* `hap.sharedRule` - if evaluated to true the `shared: true` label is used when searching, refered to as shared based search.
+* `hap.euAccessRule` - if evaluated to true the `shared: true` label is used when searching, refered to as euAccess based search.
+The configuration is done by specifying above rules in KEB helm values.
+
+Rules are independent of each other. Each plan and region combination can occur in the all rules specified above at the same time.
+
+### Format
+
+Each rule consists of a semicolon separated list of plans that the rule applies to. Additionally, a plan can be extended with `:region` suffix that makes the rule evaluate to true only if a cluster is provisioned in the specified region. List entries comply with the format `<PLAN_ID_1>:<REGION_ID_1>;<PLAN_ID_2>:<REGION_ID_2>`. Either plan or region (but never both) can be specified as wildcard `*` meaning all plans or regions should apply for specific second value. The following example lists valid and invalid configuration values:
+* `trial` - valid, rule is evaluated to true for the `trial` plan in all regions,
+* `trial:*` - valid, rule is evaluated to true for the `trial` plan in all regions,
+* `trial:eu` - valid, rule is evaluated to true for the `trial` plan in the `eu` region,
+
+* `*:eu` - valid, rule is evaluated to true for all plans in the `eu` region,
+* `eu:*` - invalid, plan must be specified in the first part of `<PLAN_ID>:<REGION_ID>` pair,
+* `*:*` - invalid, at least one of the values must be specified in the `<PLAN_ID>:<REGION_ID>` pair.
+* `*:eu;trial:eu` - valid, rule is evaluated to true for all plans in the `eu` region and for the `trial` plan in the `eu` region, configuration can be duplicated
+* `trial:eu;trial:us` - valid, rule is evaluated to true for trials plans but only in `eu` and `us` regions,
+
+### Validation
+
+Rules validation is done during the KEB startup. If the configuration is invalid, KEB will not start and an error message will be displayed in the logs. The constraints used for validation include:
+* Rules format check - all the rules must comply with the format specified above.
+* Plan existence check.
+
+### Examples
+
+The section provides configuration examples of secret bindings selection rules and shows what hyperscaler pools they correspond to.
+
+By default, KEB uses the `hyperscaler-type` label to search for secrets without applying any sufixes to the label's value. It means that without any configuration the search is done by simple hyperscaler type value implemented in [KEB](https://github.com/kyma-project/kyma-environment-broker/blob/main/common/hyperscaler/hyperscaler_type.go).
```

Comment:
It is nit-picking but KEB does not search for secrets. It searches for secrets' names.

Reply:
https://github.com/kyma-project/kyma-environment-broker/pull/1643/commits/421df91adf829e8b0ae189bf9ea004b1adb1dacc